WebApi/Controllers/ItemsController.cs

```diff
@@ -67,62 +67,62 @@ public async Task<IActionResult> CreateItem([FromForm] Item item)
         }
 
         [Authorize]
-        [HttpPut("{id}")]
-        public async Task<IActionResult> EditItem([FromRoute] Guid id, [FromForm] Item item)
+        [HttpPost("rate/{id}")]
+        public async Task<IActionResult> RateItem([FromRoute] Guid id ,[FromForm] Rating Rating)
         {
             var user = await GetUser();
-            
-            await Mediator.Send(new Edit.Command { Id = id, Item = item, User = user });
-            
+
+            await Mediator.Send(new RateItem.Command {Id = id , Rating = Rating , User = user });
+
             return NoContent();
         }
 
         [Authorize]
-        [HttpPut("{id}")]
-        public async Task<IActionResult> UnsuspendItem([FromRoute] Guid id)
+        [HttpPost("apply/{id}")]
+        public async Task<IActionResult> ApplyForItem([FromRoute] Guid id, [FromForm] Applicant applicant)
         {
             var user = await GetUser();
 
-            await Mediator.Send(new Edit.Command { Id = id , User = user});
+            await Mediator.Send(new ApplyForItem.Command { Id = id, Applicant = applicant, User = user });
 
             return NoContent();
         }
 
         [Authorize]
-        [HttpDelete("{id}")]
-        public async Task<IActionResult> DeleteItem([FromRoute] Guid id)
+        [HttpPut("{id}")]
+        public async Task<IActionResult> EditItem([FromRoute] Guid id, [FromForm] Item item)
         {
             var user = await GetUser();
             
-            await Mediator.Send(new Delete.Command { Id = id, User = user });
+            await Mediator.Send(new Edit.Command { Id = id, Item = item, User = user });
             
             return NoContent();
         }
 
         [Authorize]
-        [HttpPost("redeem/{id}")]
-        public async Task<IActionResult> RedeemItem([FromRoute] Guid id)
+        [HttpPut("unsuspend/{id}")]
+        public async Task<IActionResult> UnsuspendItem([FromRoute] Guid id)
         {
             var user = await GetUser();
 
-            await Mediator.Send(new Redeem.Command { Id = id, User = user });
-            
+            await Mediator.Send(new Edit.Command { Id = id , User = user});
+
             return NoContent();
```

Comment:
what's the difference between edititem and unsuspenditem?

Reply:
or you forgot to change it to unsuspenditem.command

Reply:
I forgot to change it